docs/user/vsphere/install.md

```diff
@@ -0,0 +1,90 @@
+# Cluster Installation
+
+At this point, you are ready to perform the OpenShift installation. You have two choices for installing your cluster on
+vSphere, Installer-Provided Infrastructure (IPI) or User-Provided Infrastructure (UPI). See below for an example of an
+IPI install.
+
+To see a guided example of a UPI install, see [Install: User-Provided Infrastructure](install_upi.md)
+
+## Example: Installer-Provided Infrastructure (IPI)
+
+The steps for performing an IPI-based install are outlined [here][cloud-install]. Following this guide you may begin at
+the Download step.
+
+### Trust Root CA
+The `create install-config`, `create cluster`, and `destroy cluster` commands access the vCenter API. The vCenter's root CA certificates must be added to the system trust before connecting to the API. The certs can be downloaded from the vCenter's homepage:
```

Comment:
>  The `create install-config`, `create cluster`, and `destroy cluster` commands access the vCenter API.

I think enumerating these commands is over stitching, there is a change any command you run will access the vCenter API to validate and prevent failures.